README.md

```diff
@@ -57,6 +56,24 @@ The services rely on a configuration file in `/etc/irods/` which dictates which
 
 Once the REST API is running install nginx and then copy `/etc/irods/irods-client-rest-cpp-reverse-proxy.conf.template` to `/etc/nginx/sites-available/irods-client-rest-cpp-reverse-proxy.conf` and then symbolically link it to `/etc/nginx/sites-enabled/irods-client-rest-cpp-reverse-proxy.conf`.   Nginx will then need to be restarted with `sudo service nginx restart`, or however your specific platform manages services.
```

Comment:
sudo service -> sudo systemctl

Reply:
possible hyphens to underscores for consistency?

Reply:
done